StatusMonitor/StatusMonitor.cs

```diff
@@ -90,56 +94,63 @@ public void start()
             {
                 Logging.Error("Directory " + Directory + " not supported: ", nsex);
             }
-            if (fileInfo.Exists)
+            if (fileInfo != null)
             {
                 try
                 {
                     using (FileStream fs = new FileStream(fileInfo.FullName, FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
                     using (StreamReader reader = new StreamReader(fs, Encoding.UTF8))
                     {
-                        lastStatus = reader.ReadLine() ?? string.Empty;
+                        lastStatusJson = reader.ReadLine() ?? string.Empty;
 
                         // Main loop
                         while (running)
                         {
-                            if (fileInfo == null)
-                            {
-                                // Status.json could not be found. Sleep until a Status.json file is found.
-                                Logging.Info("Error locating Elite Dangerous Status.json. Status monitor is not active. Have you installed and run Elite Dangerous previously? ");
-                                while (fileInfo == null)
-                                {
-                                    Thread.Sleep(5000);
-                                    fileInfo = Files.FileInfo(Directory, "Status.json");
-                                }
-                                Logging.Info("Elite Dangerous Status.json found. Status monitor activated.");
-                                return;
-                            }
-                            else
+                            if (Processes.IsEliteRunning())
```

Comment:
Consider Extract Method for the body of this `if` statement.

Reply:
Refactored.